docs/src/basics/structs_tuples_and_enums.md

```diff
@@ -73,7 +74,23 @@ enum Color {
 }
 ```
 
-Here, we have defined five potential colors. Each enum variant is just the color name. As there is no extra data associated with each variant, we say that each variant is of type `()`, or unit. It is also possible to have an enum variant contain extra data. Take a look at this more substantial example, which combines struct declarations with enum variants:
+Here, we have defined five potential colors. Each enum variant is just the color name. As there is no extra data associated with each variant, we say that each variant is of type `()`, or unit.
+
+To instantiate a variable with the value of an enum the syntax is
+
+> **Note**
+> enum instantiation does not require the `~` tilde syntax
```

Comment:
Wasn't the need for this completely removed? cc @mohammadfawaz

Reply:
Not yet. The tilde is still needed for structs. The new parser should make its removal easier.